amida-tech/indaba-client
# Feature/inba 101 reducer reorg redone

#### What's this PR do?
Breaks apart the surveys into their own reducer. This was actually not done properly before because there is a single reducer action that wasn't accounted for. It is now. 

#### Related JIRA tickets:
INBA-101.

#### How should this be manually tested?
Start 'er up. Hit up `http://localhost:3000/project` and try changing the survey status (card towards the top left). Also, go to a Task Review (say `http://localhost:3000/task-review/0/3`) and check that the survey questions displays below. 

#### Any background context you want to provide?

#### Screenshots (if appropriate):
